examples/anchor_text_movie.ipynb

```diff
@@ -17,18 +17,29 @@
   {
```

Comment:
<p>unkonw --&gt; unknown</p><p> </p><p>I would remove the last two sentences, e.g. <code>exercise</code>  is not actually in the anchor.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/432/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>